tasklist/importer-860/src/main/java/io/camunda/tasklist/zeebeimport/v860/processors/es/ProcessInstanceZeebeRecordProcessorElasticSearch.java

```diff
@@ -173,4 +185,43 @@ private boolean isOfType(
     }
     return bpmnElementType.equals(type);
   }
+
+  private UpdateRequest persistFlowNodeDataToListView(final FlowNodeInstanceEntity flowNodeInstance)
```

Comment:
No reason, UpdateRequest also creates the document if it does not exist, and in case of update, it just updates the document when exists. 
So at the end if there is an importer using this ID I can update that properly if necessary. For the Process I can use IndexRequest without ptoblems

But for UserTasks and Variables, UpdateRequest is necessary, because there are status update, and I can apply this on tasklist-list-view using the same methods.